### PR TITLE
Source Zendesk Support: update docs

### PR DESCRIPTION
**Pull request by artem1205**
## What

update docs (deleted records)

## How

update docs (deleted records)

## Recommended reading order
1. `zendesk-support.md`

## 🚨 User Impact 🚨

no breaking changes

## Pre-merge Actions

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

File: docs/integrations/sources/zendesk-support.md [hunk 1]
```diff
@@ -113,6 +112,18 @@ The Zendesk Support source connector supports the following streams:
 - [Users](https://developer.zendesk.com/api-reference/ticketing/ticket-management/incremental_exports/#incremental-user-export) \(Incremental\)
 - [UserFields](https://developer.zendesk.com/api-reference/ticketing/users/user_fields/#list-user-fields)
 
+### Deleted Records Support
+The Zendesk Support source connector fetch deleted records in streams:
```
Comment (sherifnada) [inline docs/integrations/sources/zendesk-support.md:116]:
```suggestion
The Zendesk Support connector fetches deleted records in the following streams:
```

File: docs/integrations/sources/zendesk-support.md [hunk 2]
```diff
@@ -113,6 +112,18 @@ The Zendesk Support source connector supports the following streams:
...
+
+| Stream                   | Deleted indicator field |
```
Comment (sherifnada) [inline docs/integrations/sources/zendesk-support.md:118]:
```suggestion
| Stream                   | Deletion indicator field |
```

**Review by sherifnada — APPROVED** [review]

File: docs/integrations/sources/zendesk-support.md (hunk 2 above)
Comment (artem1205) [reply docs/integrations/sources/zendesk-support.md]: updated

File: docs/integrations/sources/zendesk-support.md (hunk 1 above)
Comment (artem1205) [reply docs/integrations/sources/zendesk-support.md]: updated

**Pull request merged**